gradle/project.gradle.kts

```diff
@@ -20,25 +20,26 @@ import kotlin.text.Charsets.UTF_8
 
 // Note: the plugins block is evaluated before the script itself, so no variables can be used
 plugins {
-    kotlin("jvm") version "1.2.71"
+    kotlin("jvm") version "1.3.0"
     application
     id("com.github.ben-manes.versions") version "0.20.0"
     id("de.undercouch.download") version "3.4.3"
 }
 
 /** working directory of the application */
-val workDir = file("working dir")
+val dirProject = file("working dir")
```

Comment:
`dirProject` is a bit misleading, since it sounds like it refers to the project root, but it actually still refers to the working dir.

Reply:
Very true. dirWorking? Hm. I wanted to have all files to start with dir/file prefix to separate the variables.

Reply:
I was always unsatisfied with the word `working` tbh, but I don't know what else to call it ^^

Reply:
Yes me too. I have no word for such directory. It appears to be a standard. For now, `dirWorking` it is.